openslides/core/management/commands/getgeiss.py

```diff
@@ -0,0 +1,56 @@
+import json
+import os
+import stat
+import sys
+from urllib.request import urlopen, urlretrieve
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    """
+    Command to get the latest release of geiss from github.
+    """
+    help = 'Get the latest geiss release from github.'
+
+    def handle(self, *args, **options):
+        response = urlopen('https://api.github.com/repos/ostcar/geiss/releases/latest').read()
```

Comment:
No. I don't think its important. Please add the settings, if you think it it's worth the time.